src/main/java/com/conveyal/gtfs/error/NewGTFSError.java

```diff
@@ -111,15 +113,23 @@ public NewGTFSError setEntityId(String entityId) {
         return this;
     }
 
-    // Builder to add entity sequence info from string (values during load stage are passed in as strings from csv
-    // reader)
+    /**
+     * Builder to add entity sequence info from string (values during load stage are passed in as strings from csv
+     * reader).
+     */
     public NewGTFSError setSequence(String sequenceAsString) {
-        try {
-            // Parse int from string value found during load stage.
-            this.entitySequenceNumber = Integer.parseInt(sequenceAsString);
-        } catch (NumberFormatException e) {
-            e.printStackTrace();
+        if (sequenceAsString == null || sequenceAsString.isEmpty()) {
```

Comment:
I looked at the internal workings of `String.isEmpty` and also `String.isBlank` and I think `String.isBlank` would be a better choice here since `isEmpty` only checks if the String is a zero-length string whereas isBlank will make sure that both the string isn't zero-length, but also contains at least one non-whitespace character.

Reply:
On second thought, `isEmpty` actually seems to comply with the [GTFS spec](http://gtfs.org/reference/static/#file-requirements):

> Remove any extra spaces between fields or field names. Many parsers consider the spaces to be part of the value, which may cause errors.